functions/create-payment-intent.ts

```diff
@@ -1,18 +1,19 @@
-// An endpoint that calculates the order total and creates a 
-// PaymentIntent on Stripe 
-
+// An endpoint that calculates the order total and creates a
+// PaymentIntent on Stripe
 import { Tigris } from "@tigrisdata/core"
-import { Product } from "~/models/tigris/catalog/products"
+import { Product } from "../db/models/product"
```

Comment:
Should we also improve this so that the connection is created once in a shared library, similar to how we do it in `lib/tigris.ts` in other starters?